content/home/examples/an-application.md

```diff
@@ -4,4 +4,4 @@ order: 2
 domid: todos-example
 ---
 
-Usando `props` y `state`, podemos integrar todo en una pequeña aplicación de pendientes. Este ejemplo usa `state` para tener un control de la lista actual de objetos así como el texto que el usuario ha entrado. A pesar que los `event handlers` parezcan que fue renderizados, estos seran recolectados e implementados usando una delegación de eventos.
+Usando `props` y `state`, podemos integrar todo en una pequeña aplicación de pendientes. Este ejemplo usa `state` para tener un control de la lista actual de objetos así como el texto que el usuario ha ingresado. A pesar que los `event handlers` parezcan que fueron renderizados, estos seran recolectados e implementados usando una delegación de eventos.
```

Comment:
Isn't `aplicación de pendientes` a little bit vague? Perhaps a better form would be `aplicación de tareas pendientes`.

Also, I would write the last sentence like this:

`A pesar de que los `event handlers` parecen ser renderizados inline, éstos serán recolectados e implementados usando delegación de eventos.`

Reply:
The only thing I disagree of your rewriting of the last sequence is `éstos` because [the Spanish dictionary has stopped using it](https://www.upb.edu.co/es/blogs/palabras-en-orden/los-pronombres-demostrativos-ya-no-llevan-tilde), so it's not necessary anymore.

Reply:
I wasn't aware of that, thank you.